components/menu/index.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export {Menu} from './menu';
+export {SubMenu} from './sub_menu';
+export {MenuItem, styled} from './menu_item';
+export type {MenuItemProps} from './menu_item';
+export {MenuItemDivider} from './menu_item_divider';
```

Comment:
Even though this might cause some naming conflicts down the line, I do kind of like the idea of just exporting these as `Item`, `ItemProps`, and `Divider` to make them a bit more concise. That does also let people import them like this which could solve any naming conflicts
```js
import * as Menu from 'components/menu';

function MyMenu() {
    return (
        <Menu.Menu>
            <Menu.Item/>
            <Menu.Item/>
            <Menu.Divider/>
            <Menu.Item/>
            <Menu.Item/>
        </Menu.Menu>
    );
}
```

Come to think of it, that's more or less what React Bootstrap does except it does it in a way that the outer level is just `Menu` while the others are still prefixed like `Menu.Item`

Reply:
Yes changed because i think its more concise that way.